torchxrayvision/datasets.py

```diff
@@ -293,9 +312,9 @@ def __getitem__(self, idx):
         
         
         imgid = self.csv['Image Index'].iloc[idx]
-        img_path = os.path.join(self.imgpath, imgid)
+        #img_path = os.path.join(self.imgpath, imgid)
         #print(img_path)
-        img = imread(img_path)
+        img = self.get_image(imgid)
```

Comment:
You added this for NIH_Dataset but it doesn't extend TarDataset.

Reply:
Ah, let me fix that